engine/src/main/java/org/hibernate/search/query/grouping/SimpleGroupingResult.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.query.grouping;
+
+import java.io.IOException;
+import java.util.LinkedList;
+import java.util.List;
+
+import org.hibernate.search.query.engine.spi.DocumentExtractor;
+
+/**
+ * @author Sascha Grebe
+ */
+public class SimpleGroupingResult implements GroupingResult {
```

Comment:
Similar to the comment to SimpleGroup, I think this should be in an impl package